<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EEGHeadMeasure</class>
 <widget class="QMainWindow" name="EEGHeadMeasure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EEG Head Measure</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="0">
     <widget class="QGroupBox" name="gElectrodes">
      <property name="title">
       <string>Electrodes</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>1</number>
         </property>
         <item row="0" column="3">
          <widget class="QCheckBox" name="FP2">
           <property name="text">
            <string>FP2</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="Cz">
           <property name="text">
            <string>Cz</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="F7">
           <property name="text">
            <string>F7</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="T3">
           <property name="text">
            <string>T3</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QCheckBox" name="F8">
           <property name="text">
            <string>F8</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QCheckBox" name="F4">
           <property name="text">
            <string>F4</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="C3">
           <property name="text">
            <string>C3</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QCheckBox" name="T4">
           <property name="text">
            <string>T4</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QCheckBox" name="O1">
           <property name="text">
            <string>O1</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QCheckBox" name="O2">
           <property name="text">
            <string>O2</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="FP1">
           <property name="text">
            <string>FP1</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QCheckBox" name="C4">
           <property name="text">
            <string>C4</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="F3">
           <property name="text">
            <string>F3</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="Fz">
           <property name="text">
            <string>Fz</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QCheckBox" name="P4">
           <property name="text">
            <string>P4</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QCheckBox" name="Pz">
           <property name="text">
            <string>Pz</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="P3">
           <property name="text">
            <string>P3</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="T5">
           <property name="text">
            <string>T5</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QCheckBox" name="T8">
           <property name="text">
            <string>T8</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QCheckBox" name="O1_2">
           <property name="text">
            <string>OZ</string>
           </property>
           <property name="tristate">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget_2" native="true"/>
       </item>
       <item row="1" column="2">
        <widget class="QWidget" name="widget_3" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="m_imgHead">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="EEGHeadMeasure.qrc">:/EEGHeadMeasure/EEGImage.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="m_LongitudeLabel">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Longitude:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>cm</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>cm</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="m_Longitude">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="m_PerimeterLabel">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Perimeter:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QDoubleSpinBox" name="m_Perimeter">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Show Electrodes</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QWidget" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>m_Longitude</tabstop>
  <tabstop>m_Perimeter</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>FP1</tabstop>
  <tabstop>FP2</tabstop>
  <tabstop>F7</tabstop>
  <tabstop>F3</tabstop>
  <tabstop>Fz</tabstop>
  <tabstop>F4</tabstop>
  <tabstop>T4</tabstop>
  <tabstop>T3</tabstop>
  <tabstop>C3</tabstop>
  <tabstop>Cz</tabstop>
  <tabstop>C4</tabstop>
  <tabstop>F8</tabstop>
  <tabstop>T5</tabstop>
  <tabstop>P3</tabstop>
  <tabstop>Pz</tabstop>
  <tabstop>P4</tabstop>
  <tabstop>T8</tabstop>
  <tabstop>O1</tabstop>
  <tabstop>O2</tabstop>
 </tabstops>
 <resources>
  <include location="EEGHeadMeasure.qrc"/>
  <include location="EEGHeadMeasure.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>gElectrodes</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
